docs/how-tos/manage-fleet.md

```diff
@@ -72,13 +72,13 @@ If you'd like to look at an example, see [Monitor Air Quality with a Fleet of Se
 {{% tablestep link="/cloud/locations/" %}}
 **2. Create locations**
 
-1. Click **FLEET** in the upper-left corner of the page.
+1. Click **FLEET** in the upper-left corner of the page and click **LOCATIONS**.
    A new location called `First Location` is automatically generated for you.
 
-   Click on the location and use the **...** menu next to edit the location name to what it represents for your use case.
+   Use the **...** menu next to edit the location name to what it represents for your use case.
```

Comment:
Can't parse whether the `next` means `next to` or `use next` - but it makes sense without I think
```suggestion
   Use the **...** menu to edit the location name to what it represents for your use case.
```